lib/methods/export.js

```diff
@@ -1,3 +1,5 @@
+const _ = require('lodash');
```

Comment:
yeah I know... but no time and it does the job

Reply:
At least just import the feature you need:
```
const { uniqBy } = require('lodash');
```